code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -31,7 +31,7 @@
 	if(!lacks_power())
 		var/area/home = get_area(src)
 		if(home.powered(AREA_USAGE_EQUIP))
-			home.use_power(500 * seconds_per_tick, AREA_USAGE_EQUIP)
+			home.apc?.terminal?.use_energy(500 * seconds_per_tick, channel = AREA_USAGE_EQUIP)
```

Comment:
```suggestion
			home.apc?.terminal?.use_energy((0.5 KILO JOULES) * seconds_per_tick, channel = AREA_USAGE_EQUIP)
```

Reply:
Because we multiply this by time, it should be WATTS. Also it could just be 500 instead of 0.5 KILO.

Reply:
Also it's safe to not use parenthesis here.